PlaceholderSession/ViewModel/ViewModel.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ViewModel.swift
+//  PlaceholderSession
+//
+//  Created by Maxos on 6/6/23.
+//
+
+import Foundation
+
+protocol UsersView: AnyObject {
+    func display(_ users: [UsersModel])
+}
+
+class ViewModel {
```

Comment:
ViewModel better to rename in some consistent way with its Controller. UserViewModel maybe